install.sh

```diff
@@ -49,59 +66,46 @@ if psql -lqt | cut -d \| -f 1 | grep -qw $DBNAME; then
 
 else
 
-   #Create Postgresql Pre-reqs
-   sudo postgres <<EOF
-   psql -c "CREATE USER $DBUNAME WITH PASSWORD '$DBPASS' CREATEDB;"
-   createdb -O$DBUNAME -Eutf8 $DBNAME
-   echo "Postgres user and database '$1' created."
+#Create Postgresql Pre-reqs
+sudo su postgres <<EOF
```

Comment:
I don't think you want to run `su` in this case. running with sudo should be sufficient.

Reply:
Changed in the new script